Dockerfile

```diff
@@ -0,0 +1,11 @@
+# -----------------------------------------------------------------
+# Docker file to copy the generated binary from the `out` directory
+# -----------------------------------------------------------------
+FROM centos:7
+LABEL author Xavier Coulon <xcoulon@redhat.com>
+
+EXPOSE 8080
+ARG BINARY
+COPY ${BINARY} /usr/local/bin/fabric8-toggles-service
+
+ENTRYPOINT [ "fabric8-toggles-service" ]
```

Comment:
Have you tried to run this on a secured OS?

Reply:
ah right, I forgot the `USER` directive

Reply:
I think this should work when in /usr/local/bin as long as no disk access etc is attempted.. But not sure. Hench, have you tried? :)